mlir/include/mlir/Dialect/Rock/Generator/ConvGenerator.h

```diff
@@ -130,6 +130,13 @@ class ConvGenerator {
     std::transform(permutation.begin(), permutation.end(),
                    std::back_inserter(targetLayout),
                    [&targetSpec](int64_t p) { return targetSpec[p]; });
+
+    // +++pf:  update old key names.
```

Comment:
I used this same comment in all the places I have this kind of conversion code.  My intent is that we'll remove it as we finish the changeover.